### PR TITLE
Clean up #else in EECodeManager::FixContext

### PR DESCRIPTION
**Pull request by parjong**
It seems that only x86 ports uses EECodeManage::FixContext, but EECodeManage::FixContext contains unnecessary #else region in it.

This commit cleans up unnecessary #else region from EECodeManage::FixContext.

### CODE REVIEW

**Comment by parjong** [comment]
@jkotas @janvorli Please take a look.

File: src/inc/eetwain.h [hunk 1]
```diff
@@ -161,7 +161,7 @@ enum
 };
 
 #ifndef DACCESS_COMPILE
-
+#ifdef _TARGET_X86_
```
Comment (jkotas) [inline src/inc/eetwain.h]: Should this rather be WIN64EXCEPTIONS like in the other change?

Comment (parjong) [inline src/inc/eetwain.h]: @jkotas The existing implementation uses ``_TARGET_X86_``, and thus I uses ``_TARGET_X86_`` here. Is this related with ``WIN64EXCEPTIONS``?? If so, it would be better to use ``WIN64EXCEPTIONS`` as you mentioned.

Comment (jkotas) [inline src/inc/eetwain.h]: This method does not make sense for !WIN64EXCEPTIONS . I think it would more appropriate to put it under `#ifndef WIN64EXCEPTIONS`.

Comment (parjong) [inline src/inc/eetwain.h]: Do you mean that this method is related with ``WIN64EXCEPTIONS`` (not ``_TARGET_X86_``)?

Comment (jkotas) [inline src/inc/eetwain.h]: Correct. It is only called in src/vm/i386/excepx86.cpp that is only included for !WIN64EXCEPTIONS. You should not need it for Linux/x86.

**Comment by parjong** [comment]
@dotnet-bot test Ubuntu x64 Checked Build and Test please
